src/gluonts/model/rotbaum/_model.py

```diff
@@ -124,29 +125,50 @@ def _create_xgboost_model(model_params: Optional[dict] = None):
             }
         return xgboost.sklearn.XGBModel(**model_params)
 
-    def fit(self, x_train, y_train, max_sample_size=None, seed=1, **kwargs):
+    def fit(
```

Comment:
Can we add type annotations?

Reply:
Done.

Reply:
Sorry, maybe I was not clear. I think type annotations in the function definition are better than in the doc string and as far as I understand, this is what we converged to in GluonTS.